File: idea-plugin/src/main/java/org/ifinalframework/jetbrains/plugins/aio/api/yapi/YapiProperties.java
package org.ifinalframework.jetbrains.plugins.aio.api.yapi;


import lombok.Getter;
import lombok.Setter;
import org.springframework.boot.context.properties.ConfigurationProperties;

/**
 * YapiProperties
 *
 * @author iimik
 * @since 0.0.1
 **/
@Setter
@Getter
@ConfigurationProperties(prefix = "final.api.yapi")
public class YapiProperties {
    private String serverUrl;
    private String token;
}

File: idea-plugin/src/main/kotlin/org/ifinalframework/plugins/jetbrains/aio/api/yapi/YapiProperties.kt
package org.ifinalframework.plugins.jetbrains.aio.api.yapi

import org.springframework.boot.context.properties.ConfigurationProperties


/**
 * YapiProperties
 *
 * @author iimik
 * @since 1.6.0
 **/
@ConfigurationProperties("final.api.yapi")
data class YapiProperties(
    val serverUrl: String?,
    val token: String?,
)
